src/styles/styles.js

```diff
@@ -2874,6 +2874,37 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    joinRoomPromptContainer: {
+        maxHeight: 92,
+        height: 92,
+        borderRadius: 18,
+        padding: 20,
+        backgroundColor: themeColors.overlay,
+        display: 'flex',
+        flexDirection: 'row',
+    },
+
+    joinRoomPromptTextContainer: {
+        paddingTop: 3.5,
+        paddingBottom: 3.5,
+    },
+
+    joinRoomPromptButtonContainer: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'flex-end',
+        alignItems: 'center',
+    },
+
+    joinRoomButton: {
+        maxHeight: 40,
+        margin: 4,
+    },
+
+    joinRoomButtonText: {
+        fontSize: 13,
```

Comment:
Why do we need this here? Our default sized buttons should already have this, we shouldn't add special styles for buttons - everything would be styled at the global button level.

Reply:
The default sized buttons have a larger font size (I believe it was 15?).

Are there styles anywhere that let us use size 13 font?

Reply:
Small buttons that are 28px tall should use 11px font size (micro)

Default buttons that are 40px tall should use 13px font size (label)

Large buttons that are 52px tall should use 15px font size (default)

Reply:
If it turns out that the default button isn't using the right font size, let's change it globally.

Reply:
Ahhh gotcha, I must have misread the styles when I was originally implementing this, I'll take another look, thanks!

Reply:
Fixed this!